datasets/cifar10.py

```diff
@@ -22,7 +21,7 @@ class Data:
 
         def __init__(self, x, l, logit, flip, dequantize, rng):
 
-            D = x.shape[1] / 3                                 # number of pixels
+            D = int(x.shape[1] / 3)                                 # number of pixels
```

Comment:
Please reduce whitespace to make "# number of pixels" align with the comments below it